File: co.edu.uniandes.fixml.toC++/test/text2.xml/Instrmt.cpp
#include <iostream> 

using namespace std;

class Instrmt {

	private:
  	string Sym_;
  	string ID_;
  	string IDSrc_;
  	string CFI_;
  	string MMY_;
  	string Mat_;
  	string Strk_;
  	string StrkCcy_;
  	string Mult_;
 	
 	public: 

  	// default constructor
	Instrmt () {
		Sym_ = "AOL";
		ID_ = "KW";
		IDSrc_ = "3";
		CFI_ = "OCASPS";
		MMY_ = "20031122";
		Mat_ = "2003-11-22T00:00:00";
		Strk_ = "47.50";
		StrkCcy_ = "USD";
		Mult_ = "100";
  	
  	}
  
  	// by parameters
  	Instrmt (		string Sym_p, string ID_p, string IDSrc_p, string CFI_p, string MMY_p, string Mat_p, string Strk_p, string StrkCcy_p, string Mult_p){
	  
		Sym_ = Sym_p;
		ID_ = ID_p;
		IDSrc_ = IDSrc_p;
		CFI_ = CFI_p;
		MMY_ = MMY_p;
		Mat_ = Mat_p;
		Strk_ = Strk_p;
		StrkCcy_ = StrkCcy_p;
		Mult_ = Mult_p;
	  
  	}
};

File: co.edu.uniandes.fixml.toCSharp/test/test2.xml/Instrmt.cs


// Class for tag Instrmt
class Instrmt {

	string _CFI;
	string _ID;
	string _IDSrc;
	string _MMY;
	string _Mat;
	string _Mult;
	string _Strk;
	string _StrkCcy;
	string _Sym;
	
	
	// default constructor
	
	public Instrmt ( ) {   
		this._CFI = "OCASPS";
		this._ID = "KW";
		this._IDSrc = "J";
		this._MMY = "20031122";
		this._Mat = "2003-11-22T00:00:00";
		this._Mult = "100";
		this._Strk = "47.50";
		this._StrkCcy = "USD";
		this._Sym = "AOL";
		
		
	}
	
	public Instrmt ( string CFI, string ID, string IDSrc, string MMY, string Mat, string Mult, string Strk, string StrkCcy, string Sym ) {
		this._CFI = CFI;
		this._ID = ID;
		this._IDSrc = IDSrc;
		this._MMY = MMY;
		this._Mat = Mat;
		this._Mult = Mult;
		this._Strk = Strk;
		this._StrkCcy = StrkCcy;
		this._Sym = Sym;
		
	}
	
	
} 